<ui version="4.0" >
 <class>ChildrenEditorBase</class>
 <widget class="QWidget" name="ChildrenEditorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Children Editor</string>
  </property>
  <property name="toolTip" >
   <string>Allows you to change the order of child options.</string>
  </property>
  <property name="statusTip" >
   <string>Children Editor</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="3" row="0" column="0" >
    <widget class="QListView" name="mChildrenView" />
   </item>
   <item row="1" column="1" >
    <widget class="QToolButton" name="mDownBtn" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="mUpBtn" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
